elephant/spike_train_synchrony.py

```diff
@@ -191,8 +194,12 @@ def spike_contrast(spiketrains, t_start=None, t_stop=None,
 
     try:
         isi_min = min(np.diff(st).min() for st in spiketrains if len(st) > 1)
-    except TypeError:
-        raise ValueError("All input spiketrains contain no more than 1 spike.")
+    except:
+        # Do not raise an error but just a warning and continue the code which will return nan-values as synchrony.
+        # This is useful when running spike-contrast in a batch script, where it is necessary 
+        # to get a complete data structure (trace for all bin-sizes) also for empty spike trains. 
+        warnings.warn("All input spiketrains contain no more than 1 spike.")
+        isi_min = 0  # continue with isi_min = 0, causing to return spike-contrast with nan as synchrony values but 
     bin_min = max(isi_min / 2, min_bin)
```

Comment:
```suggestion
    if n_spikes_total == 0 :
        warnings.warn("All input spike trains contain no spikes.")
        isi_min = 0
    else:
        isi_min = min(np.diff(st).min() for st in spiketrains if len(st) > 1)

    bin_min = max(isi_min / 2, min_bin)
```

Reply:
Hello @Moritz-Alexander-Kern

Thank you for your suggestions! 
The result for the empty spike trains, looks good! 

Regarding "stationarity" in this line:
https://github.com/NeuralEnsemble/elephant/blob/fe5053a52a3aaa5c86d9a9c61f72de14b364df1a/elephant/spike_train_synchrony.py#L193
You are right, (non-)stationarity is not related how we should handle the case of a single spike. It just explains, why we can get different synchrony values even if we just have one spike per spike train.
For example, these two spike trains (non-stationary)
```
spiketrains = [
    neo.SpikeTrain([1.0]*ms, t_stop=1000 * ms),
    neo.SpikeTrain([1.2]*ms, t_stop=1000 * ms),
]
```
will lead to another synchrony value compared to these two spike trains (stationary):
```
spiketrains = [
    neo.SpikeTrain([250.0]*ms, t_stop=1000 * ms),
    neo.SpikeTrain([750.0]*ms, t_stop=1000 * ms),
]
```

Since it is possible to obtain a synchrony value even with just a single spike per spike train, I would suggest, to handle this case the same way as you suggested for the empty spiketrain by setting isi_min = 0 plus the warning (If I'm not wrong, we could use my original suggestion with the try-except block?).

Just to explain, what happens if we set isi_min = 0:
Spike-contrast calculates a synchrony for many different bin sizes ("bin size sweep") and at the end selects the optimum as the final synchrony.
The smallest ISI from all spiketrains is used to define the lower limit for the bin-size sweep. If isi_min=0, the lower limit will be set to min_bin:
https://github.com/NeuralEnsemble/elephant/blob/fe5053a52a3aaa5c86d9a9c61f72de14b364df1a/elephant/spike_train_synchrony.py#L196
So, even in the case of just one spike per spike train, the algorithm can calculate a synchrony.